2_process/src/model_diagnostics.R

```diff
@@ -0,0 +1,43 @@
+
+
+## preliminaries 
+library(dplyr)
+library(readr)
+library(stringr)
+library(whisker)
+
+## function that produces model diagnostics
+### default saves output to '2_process/out/model_diagnostic_text.txt
+
+get_diagnostics<-function(input_path = '2_process/out/model_summary_results.csv', output_path = '2_process/out/model_diagnostic_text.txt'){
+  
+  # read in processed data
+  input_data<-readr::read_csv(input_path)
+  
+  ## extract model diagnostics
+  render_data <- list(pgdl_980mean = filter(input_data, model_type == 'pgdl', exper_id == "similar_980") %>% pull(rmse) %>% mean %>% round(2),
+                      dl_980mean = filter(eval_data, model_type == 'dl', exper_id == "similar_980") %>% pull(rmse) %>% mean %>% round(2),
```

Comment:
Looks like you'll have `input_data`, but `eval_data` won't be available to the scope of this function. Perhaps change input_data <- readr... to eval_data, or find replace `eval_data`?